signature/cose/envelope.go

```diff
@@ -17,10 +19,28 @@ import (
 // MediaTypeEnvelope is the COSE signature envelope blob mediaType.
 const MediaTypeEnvelope = "application/cose"
 
+var (
+	// encOpts is the encoding options used in Sign
+	encOpts cbor.EncOptions
+
+	// decOpts is the decoding options used in Content
+	decOpts cbor.DecOptions
+)
```

Comment:
Instead of `encOpts` and `decOpts`, should we have `encMode` and `decMode` like in [go-cose](https://github.com/veraison/go-cose/blob/main/cbor.go)?